notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb

```diff
@@ -755,7 +685,7 @@
         "\n",
         "Next, upload the model's evaluation from the custom training job to the corresponding entry in the Vertex AI Model Registry.\n",
         "\n",
-        "Currently, there is not yet support for this method in the SDK. Instead, you use the lower level GAPIC API interface."
+        "Currently, there is no support for this method yet in the Python SDK. Instead, you use the lower level GAPIC API interface."
```

Comment:
change:  "Currently, there is no support for this method yet in the Python SDK"
to: " "Currently, the Vertex AI Python SDK not support for this method."